aspnetcore/toc.yml

```diff
@@ -425,6 +425,8 @@
       uid: web-api/advanced/analyzers
     - name: Conventions
       uid: web-api/advanced/conventions
+    - name: Use the HTTP REPL tool
```

Comment:
Not sure I like this title. How about **Testing APIs with dotnet httprepl**? @scottaddie ?

Reply:
I like your proposed title better. I'll incorporate this change.